src/components/divider/Divider.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+export interface Props {
+  vertical?: boolean;
+}
+
+defineOptions({
+  name: 'RuiDivider',
+});
+
+withDefaults(defineProps<Props>(), {
+  vertical: false,
+});
+</script>
+
+<template>
+  <div
+    :class="vertical ? 'border-l' : 'border-t'"
+    class="dark:border-rui-grey-800"
```

Comment:
what will be the colour in light theme mode?

can we also make the colour a prop (with a default to the common colour), just so it is easier to customise anywhere/anytime

Reply:
I make the default on the light `border-rui-grey-200`

and also make it into utility class, so it can be used by using the class.
for the custom color, we can just pass the class (I'm not sure we use other color for the border)